docs/hsi.md

```diff
@@ -454,11 +455,12 @@ This would allow subverting the Secure Boot protection, and also invalidate any
 - `not-valid`: could not determine state
 - `not-enabled`: was not enabled
 
-To meet HSI-3 on systems that run this test, the result must be `enabled`. *[v1.5.0]*
+To meet HSI-3 on systems that run this test, the result must be `enabled`. *[v1.8.0]*
```

Comment:
I think we need to capture that the attribute existed in 1.5.0 for Intel and 1.8.0 for AMD.

Reply:
fixed